definitions/mobile.d.ts

```diff
@@ -19,7 +19,6 @@ declare module Mobile {
 		sync(localToDevicePaths: ILocalToDevicePathData[], appIdentifier: IAppIdentifier, liveSyncUrl: string, options: ISyncOptions): IFuture<void>;
 		debug(packageFile: string, packageName: string): IFuture<void>;
 		openDeviceLogStream(): void;
-		getInstalledApplications(): IFuture<string[]>;
```

Comment:
Yes, it was a duplicated declaration of a method. The same getInstalledApplciations() is available  at the start of the class declaration.